core/main/src/firebolt/rpc_router.rs

```diff
@@ -133,6 +133,7 @@ async fn resolve_route(
         let rpc_header = get_rpc_header(&req);
         let protocol = req.ctx.protocol.clone();
         let request_id = req.ctx.request_id;
+
```

Comment:
@coderabbitai write as many unit tests as possible for this file

Reply:
@coderabbitai: these unit tests do not compile. Try again, and focus on getting the unit tests to compile